projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -1592,6 +1592,36 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
         return this._columnPipeArgs;
     }
 
+    /**
+     * @remarks
+     * Pass optional parameters to control properties of the default column editors.
+     * Accepts an `IColumnEditorOptions` object with the `dateTimeFormat` property.
+     * For more details see https://angular.io/api/common/DatePipe
+     * @example
+     * ```typescript
+     * const editorOptions: IColumnEditorOptions = {
+     *      dateTimeFormat: 'MM/dd/YYYY',
+     * }
+     * ```
+     * ```html
+     * <igx-column dataType="date" [editorOptions]="editorOptions"></igx-column>
+     * ```
+     * @memberof IgxColumnComponent
+     */
+    @notifyChanges()
+    @WatchColumnChanges()
+    @Input()
+    public set editorOptions(value: IColumnEditorOptions) {
+        this._editorOptions = value;
+        this.grid.pipeTrigger++;
+    }
+    public get editorOptions(): IColumnEditorOptions {
+        if (!this._editorOptions?.dateTimeFormat) {
+            return { ...this._editorOptions, dateTimeFormat: this.pipeArgs.format };
```

Comment:
Any chance this can be centralized somewhere as a getter or check in the logic that's about to pass the format to the UI components?